publish

```diff
@@ -8,6 +8,11 @@ if [ "x" = "x$VERSION" ]; then
   exit 1
 fi
 
+if [ "x-h" = "x$VERSION" ]; then
+  echo 'usage: publish [-h] version'
+  exit 0
+fi
```

Comment:
maybe this should be the output of just running `./publish` as well

Reply:
That's a good suggestion, but I like that they exit with different codes, indicating that one is an error case and one is an expected non-error case